sorting_methods.py

```diff
@@ -0,0 +1,29 @@
+def bubble_up(liczby):
+    num = liczby[1]
+    ind = liczby[0]
+    n = len(num)
+    zmieniona = True
+    while zmieniona == True:
+        zmieniona = False
+        for i in range(n-1):
+            if num[i] < num[i+1]:
+                num[i], num[i+1] = num[i+1],num[i]
+                ind[i], ind[i + 1] = ind[i + 1], ind[i]
+                zmieniona = True
+    return num,ind
+
+def Insertion_up(liczby):
```

Comment:
bubble_up, insertion_up . Poczytaj clean code. To ani nie są dobre nazwy funkcji ani funkcje nie powinny mieć tego zaszytego w nazwie(kierunku). Albo zakładasz że zawsze sortujesz rosnąco albo dajesz opcję wyboru i funkcja przyjmuje parametr, który nam mówi w którą stronę sortujemy. Jeśli dobrze w ogóle zrozumiałem zamysł _up. Nie musisz odpisywać, sam przemyśl.